modules/rostests/apitests/win32nt/CMakeLists.txt

```diff
@@ -64,47 +64,22 @@ list(APPEND SOURCE
     #osver.c
     win32nt.h)
 
-add_library(win32knt_static STATIC ${SOURCE} w32knapi.rc)
-add_dependencies(win32knt_static xdk)
-add_pch(win32knt_static win32nt.h SOURCE)
+add_executable(win32knt_apitest
+    ${SOURCE}
+    testlist.c
+    w32knapi.rc)
 
-set(WIN32KNT_IMPORTLIBS gdi32 user32 shell32 advapi32 msvcrt kernel32 ntdll)
```

Comment:
So the idea there is that there is basically no point in having win32k tests that cover Windows versions that we don't support? (basically everything below Windows Server 2003 SP2)

Reply:
The idea is to always link to win32u.dll using delayimport. If a function doesn't exist in a DLL, that test will abort. On Windows 10 win32u.dll is part of the OS, on other OSes, you need to copy the correct version as win32u.dll. This way there is no need to hack the CMakeLists file.